TrackerSB/IEEEAITestChallenge2021
# PR #01: Test Case for Scenario 02

Changes:
- Adding lgsvl module to python_src folder
- Adding a test case for Scenario 02
- Adding a sample file with comments on how to connect to simulator, place vehicles, and adjust their properties

Steps to replicate the test case on the localhost:
1. Run the following command from `python_src/` to install Python files and necessary dependencies: `pip3 install --user -e .`
2. Now launch the simulator (either binary .exe file or from Unity Editor).
3. Click the Open Browser button to open the Simulator UI.
4. After the default maps and vehicles have been downloaded, navigate to the Simulations tab.
5. Create a new Simulation. Give it a name and check the API Only option. Click Submit.
6. Select the newly created Simulation and click the "Play" button in the bottom right.
7. Execute `python_src/source/tc_02.py` from PyCharm
